.gitignore

```
@@ -0,0 +1,4 @@
+
+.DS_Store
+Simulator 2.x/.DS_Store
+Simulator 2.x/extended version/.DS_Store
```

```suggestion
.DS_Store
```
Die Zeile ist ausreichend. Wird nur ein Dateiname angegeben, dann wird der in sämtlichen Verzeichnissen ausgeschlossen.